agent/consul/state/config_entry_test.go

```diff
@@ -310,6 +409,440 @@ func TestStore_ConfigEntries(t *testing.T) {
 		Protocol: "tcp",
 	}))
 	require.True(t, watchFired(ws))
+
+}
+
+func TestStore_ServiceDefaults_Kind_Destination(t *testing.T) {
```

Comment:
I should probably do an actual diff, but just looking at `TestStore_ServiceDefaults_isDestination` above and this function, I didn't see the difference.